platforms/ios/lib/WysiwygComposer/Sources/WysiwygComposer/Components/WysiwygComposerView/WysiwygComposerViewModel.swift

```diff
@@ -62,12 +62,12 @@ public class WysiwygComposerViewModel: WysiwygComposerViewModelProtocol, Observa
 
     /// The composer content for plain text mode.
     public var plainTextModeContent: WysiwygComposerContent {
-        // TODO: convert plain text to HTML
-        WysiwygComposerContent(plainText: plainText,
-                               html: "",
-                               attributed: NSAttributedString(string: plainText),
-                               attributedSelection: .init(location: plainText.utf16Length,
-                                                          length: 0))
+        _ = model.setContentFromMarkdown(markdown: plainText)
```

Comment:
maybe we could add `@discardableResult` to make this line a bit nicer to read?

Reply:
I don't think we can, because this is a generated func from UniFFI